ui/src/components/index.js

```diff
@@ -7,6 +7,7 @@ export { DocumentAuthors } from './Document';
 export { EmailLink } from './EmailLink';
 export { EmptyMessage } from './EmptyMessage';
 export { Error } from './Error';
+export { Forbidden } from './Forbidden/Forbidden';
```

Comment:
Since we export it from `index.js` it can be just `export { Forbidden } from './Forbidden';` like the rest